onetl/file/limit/limits_reached.py

```diff
@@ -18,18 +18,44 @@
 from typing import Iterable
 
 from onetl.base import BaseFileLimit
-from onetl.impl import RemotePath
 
 log = logging.getLogger(__name__)
 
 
-def limits_reached(limits: Iterable[BaseFileLimit], path: RemotePath) -> bool:
-    if not limits:
-        return False
+def limits_reached(limits: Iterable[BaseFileLimit]) -> bool:
+    """
+    Check if any of limits reached.
 
+    Parameters
+    ----------
+    limits : Iterable of :obj:`onetl.base.base_file_limit.BaseFileLimit`
+        Limits to test.
+
+    Returns
+    -------
+    ``True`` if any of limits is reached, ``False`` otherwise.
+
+    If no limits are passed, returns ``False``.
+
+    Examples
+    --------
+
+    .. code:: python
+
+        from onetl.file.limit import MaxFilesCount, limits_reached, limits_stop_at
+        from onetl.impl import LocalPath
+
+        limits = [MaxFilesCount(2)]
+        assert not limits_reached(limits)
+
+        assert not limits_stop_at(LocalPath("/path/to/file.csv"), limits)
+        assert limits_stop_at(LocalPath("/path/to/file.csv"), limits)
+
+        assert limits_reached(limits)
+    """
     reached = []
     for limit in limits:
-        if limit.stops_at(path):
+        if limit.is_reached:
```

Comment:
If the list of all reached limits is needed only for debug, maybe should adding `break` on the first trigger, so that it is similar to the `any` function

Reply:
Do you mean something like?
```python
debug = log.isEnabledFor(logging.DEBUG)

reached = []
for limit in limits:
  if limit.is_reached:
    if not debug:
      return True
    reached.append(limit)

if reached:
  if debug:
    log.debug(...)
  return True

return False
```

Reply:
yes

Reply:
Fixed